src/Opdex.Platform.Infrastructure/Clients/CirrusFullNodeApi/Handlers/Tokens/CallCirrusGetInterfluxTokenContractSummaryQueryHandler.cs

```diff
@@ -23,31 +24,24 @@ public CallCirrusGetInterfluxTokenContractSummaryQueryHandler(ISmartContractsMod
 
     public async Task<InterfluxTokenContractSummary> Handle(CallCirrusGetInterfluxTokenContractSummaryQuery request, CancellationToken cancellationToken)
     {
-        try
-        {
-            var ownerRequest = new LocalCallRequestDto(request.Token, request.Token,
-                $"get_{InterfluxTokenConstants.Properties.Owner}", request.BlockHeight);
-            var ownerResponse = await _smartContractsModule.LocalCallAsync(ownerRequest, cancellationToken);
-            var owner = ownerResponse.DeserializeValue<Address>();
+        var ownerRequest = new LocalCallRequestDto(request.Token, request.Token,
+            $"get_{InterfluxTokenConstants.Properties.Owner}", request.BlockHeight);
+        var ownerResponse = await _smartContractsModule.LocalCallAsync(ownerRequest, cancellationToken);
+        if (!ownerResponse.TryDeserializeValue<Address>(out var owner)) return null;
 
-            var nativeChainRequest = new LocalCallRequestDto(request.Token, request.Token,
-                $"get_{InterfluxTokenConstants.Properties.NativeChain}", request.BlockHeight);
-            var nativeChainResponse = await _smartContractsModule.LocalCallAsync(nativeChainRequest, cancellationToken);
-            var nativeChain = nativeChainResponse.DeserializeValue<ExternalChainType>();
+        var nativeChainRequest = new LocalCallRequestDto(request.Token, request.Token,
+            $"get_{InterfluxTokenConstants.Properties.NativeChain}", request.BlockHeight);
+        var nativeChainResponse = await _smartContractsModule.LocalCallAsync(nativeChainRequest, cancellationToken);
+        if (!nativeChainResponse.TryDeserializeValue<ExternalChainType>(out var nativeChain)) return null;
 
-            var nativeAddressRequest = new LocalCallRequestDto(request.Token, request.Token,
-                $"get_{InterfluxTokenConstants.Properties.NativeAddress}", request.BlockHeight);
-            var nativeAddressResponse =
-                await _smartContractsModule.LocalCallAsync(nativeAddressRequest, cancellationToken);
-            var nativeAddress = nativeAddressResponse.DeserializeValue<string>();
+        var nativeAddressRequest = new LocalCallRequestDto(request.Token, request.Token,
+            $"get_{InterfluxTokenConstants.Properties.NativeAddress}", request.BlockHeight);
+        var nativeAddressResponse =
+            await _smartContractsModule.LocalCallAsync(nativeAddressRequest, cancellationToken);
+        if (!nativeAddressResponse.TryDeserializeValue<string>(out var nativeAddress)) return null;
```

Comment:
Null is ok here

Reply:
Yes, this will only evaluate to false if the value cannot be deserialized. When `TryDeserializeValue` is called trying to deserialise to a reference type, `null` is a valid value and it'll return true.